Monitoring.NuGet/Monitoring.NuGet.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Monitoring.NuGet.csproj can be deleted from the repository

Reply:
It is deleted, see Aspects.sln changes

Reply:
Deleting it also physically